docs/voyageai.md

```diff
@@ -0,0 +1,182 @@
+# Use pgai with Voyage AI
+
+This page shows you how to:
+
+- [Configure pgai for Voyage AI](#configure-pgai-for-voyage-ai)
+- [Add AI functionality to your database](#usage)
+- [Follow advanced AI examples](#advanced-examples)
+
+## Configure pgai for Voyage AI
+
+Most pgai functions require a [Voyage AI API key](https://docs.voyageai.com/docs/api-key-and-installation#authentication-with-api-keys).
```

Comment:
nit: We should reword this sentence (and the corresponding ones in the other docs). When it was first authored, we only supported OpenAI, so "most pgai functions" DID require an OpenAI key. This sentence was copy/pasted around. Now, most **_OpenAI_** functions require an openai API key, but MOST pgai functions do not. Same goes for VoyageAI and all the other providers.

Reply:
I will take this up as a follow-on task.